TritonSE/USHS-Housing-Portal
# Linting / Formatting

## Tracking Info

Resolves #8 

## Changes

- Add linting and formatting following [linters repo](https://github.com/TritonSE/linters) (thanks @justinyaodu!)
- Add github action

## Testing

- Ensure package.json scripts work locally
- Ensure github action works

## Confirmation of Change



N/A
